incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -193,6 +195,10 @@ func (r *HierarchyConfigReconciler) syncWithForest(log logr.Logger, nsInst *core
 	hadCrit := ns.HasLocalCritCondition()
 	ns.ClearLocalConditions()
 
+	// Set external tree labels in the forest if this is an external namespace. It skips some of the
+	// syncs below accordingly.
```

Comment:
Maybe take out the second sentence? At the top level of this function, nothing gets skipped, and I was a bit confused at first.

Reply:
sg